test/validation/api/crypto/util.h

```diff
@@ -20,6 +20,7 @@ struct suite_context_s {
 	odp_queue_t queue;
 	odp_queue_type_t q_type;
 	odp_event_t (*compl_queue_deq)(void);
+	int quick_test;
```

Comment:
Perhaps this could be `partial_test` or something similar related to scope instead of speed (same change for the new function).